src/world/Macros/ScriptMacros.hpp

```diff
@@ -5,6 +5,12 @@ This file is released under the MIT license. See README-MIT for more information
 
 #pragma once
 
+/// Coords Creature
+#define MAP_CREATURE_NEAREST_COORDS GetMapMgr()->GetInterface()->GetCreatureNearestCoords
```

Comment:
Rename to MAP_GET_CREATURE_NEAREST_COORDS

Reply:
@Appled ok. about the method they are equal 

> MAP_GET_CREATURE_NEAREST_COORDS = getNearestCreature();

Reply:
look here =)